### PR TITLE
Dong outfolow condition

### PR DESCRIPTION
**Pull request by adampep**

### CODE REVIEW

**Comment by stgeke** [comment]
Can you please switch back to PnPn.

-----Original message-----

**Comment by adampep** [comment]
the error message from nek for sym bc in chkbc states
Requires PN/PN-2 STRESS FORMULATION
is this not correct?

File: turbJet/turbJet/turbJet.usr [hunk 1]
```diff
@@ -88,12 +76,29 @@ C-----------------------------------------------------------------------
...
       bID = boundaryIDList(iside,e)
- 
-      if (bID.eq.3 .or. bID.eq.4) then ! coflow + lateral
+      
+c     Dong boundary condition
+      pa = 0.0
+      U0 = 1.0                  ! characteristic velocity
+      delta = 0.1               ! small positive constant
+      if (cbu.eq.'o  ') then
+         if (iside.eq.1.or.iside.eq.3) then
```
Comment (stgeke) [inline turbJet/turbJet/turbJet.usr]: Can you please explain what this does?

**Comment by stgeke** [comment]
> the error message from nek for sym bc in chkbc states
> Requires PN/PN-2 STRESS FORMULATION
> is this not correct?

Are you sure you are using the latest master?

File: turbJet/turbJet/turbJet.usr (hunk 1 above)
Comment (adampep) [reply turbJet/turbJet/turbJet.usr]: you have to calculate normal velocity component on the outflow only, so you have to identify outflow, and the position in un[xyz] array

**Comment by adampep** [comment]
By the way. It seems right now turb_outflow is used only in mv_wall. In expansion it is commented

File: turbJet/turbJet/turbJet.usr (hunk 1 above)
Comment (stgeke) [reply turbJet/turbJet/turbJet.usr]: I don't like "heavy" code like this in the usr file. May be we should compute the face normal in FACEIS() and FACEIV() and store it in NEKUSE variable. Another option could be to introduce a subroutine to compute the local face normal like fnormal(un,ix,iy,iz,iel,iface). May be it exisits already?

Probably useful

Comment (dshaver-ANL) [reply turbJet/turbJet/turbJet.usr]: It does already exist, it's getSnormal(un,ix,iy,iz,iel,iface).

**Comment by stgeke** [comment]
Will open a new PR and push an updated version.

**Pull request closed without merging**